### DISCUSSION TITLE
Plan to update Grafana Dashboards for the Angular Deprecation?

### DISCUSSION THREAD

**Discussion by jeffmccune** [prometheus-operator/kube-prometheus#2403] [General]
Curious if there's a plan to replace the kubernetes dashboards that are shipped by default with ones compatible with Grafana 11 which will be released soon.

Currently with Grafana 10 the kubernetes dashboards display many warnings about the Angular v1 deprecation which will move to removal by default in Grafana 11 set for May 2024.

FYI: [Removal of AngularJS support in Grafana: what you need to know](https://grafana.com/blog/2024/03/11/removal-of-angularjs-support-in-grafana-what-you-need-to-know/)

**Comment by kakkoyun** [comment]
I don't think nobody would disagree with updating them. Just someone needs to do the work. 
This is a matter of upgrading jsonnet dependencies (since we use upstream Grafana jsonnet libs to generate dashboards) and regenerating.

Contributions are most welcome!

**Reply by Vinaum8** [reply]
@kakkoyun how i could help?

**Reply by kakkoyun** [reply]
@Vinaum8 If you have experience with jsonnet. Maybe you can give us a hand. I'd be happy to guide you.

**Reply by Vinaum8** [reply]
I have no experience with jsonnet. Normally, to share a dashboard I make the changes in "clickops" and then export the dashboard as json. Would this work?

**Reply by kakkoyun** [reply]
Unfortunately, it wouldn't work. We generate everything using jsonnet and keep them in the repo checked in.

**Comment by silkyroadsilk** [comment]
Any updates on this in 2025? Any plans to update dashboards?

**Reply by Vinaum8** [reply]
I am using Grafana version 11.3.1 and the migration warnings are gone.

```
image:
      registry: docker.io
      repository: grafana/grafana
      tag: 11.3.1
```
